src/styles/main.scss

```diff
@@ -1,3 +1,57 @@
-body {
+@import './variables.scss';
+* {
+  box-sizing: border-box;
   margin: 0;
 }
+
+.calendar {
+  padding: 25px 10px 10px;
+
+  &__field{
+    display: flex;
+    flex-wrap: wrap;
+    gap: $cells-gap;
+    width: (7 * $cell-size) + (6 * $cells-gap);
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    height: 100px;
```

Comment:
The height of the day cells should be equal to the value of $cell-size variable. Replace 100px with $cell-size.